tests/test_cgnsutilities.py

```diff
@@ -10,15 +11,34 @@ class TestGrid(unittest.TestCase):
     def setUp(self):
         self.grid = readGrid(os.path.abspath(os.path.join(baseDir, "../examples/717_wl_L2.cgns")))
 
-    def test_getTotalCellsNodes(self):
+    def test_getTotalCellsNodes(self, train=False):
         totalCells, totalNodes = self.grid.getTotalCellsNodes()
-        self.assertEqual(totalCells, 15120)
-        self.assertEqual(totalNodes, 18753)
+        refFile = os.path.join(baseDir, "ref", "totalCellsNodes.ref")
+        with BaseRegTest(refFile, train=train) as handler:
+            handler.root_add_val("Total cells", totalCells, tol=0)
+            handler.root_add_val("Total nodes", totalNodes, tol=0)
 
-    def test_getWallCellsNodes(self):
+    def train_getTotalCellsNodes(self):
+        self.test_getTotalCellsNodes(train=True)
+
+    def test_getWallCellsNodes(self, train=False):
```

Comment:
I'm not a fan of this design pattern. I'd rather have just one test, but that seem like a bigger issue than just this PR

Reply:
I think it makes sense to have separate tests for each function because then it is easy to see what is broken if the test fails. I don't like having a separate train function and ref file for each test though.